charts/datahub/subcharts/acryl-datahub-actions/values.yaml

```diff
@@ -34,6 +34,8 @@ service:
   type: ClusterIP
   port: 9093
 
+# Extra labels for Deployment
+extraLabels: {}
```

Comment:
Please add some example of how to specify extra labels. Commented out of course. This should apply to all sub-chart values.yaml

Reply:
Will do!